src/Headers/Particle.h

```diff
@@ -240,6 +240,31 @@ struct SphParticle : public Particle<ndim>
     alpha    = (FLOAT) 0.0;
     dalphadt = (FLOAT) 0.0;
   }
+
+  class DensityParticle {
+  public:
+    DensityParticle() : m(0), u(0), gpot(0), ptype(0) {} ;
+    DensityParticle(const SphParticle<ndim>&p) {
+      for (int i=0; i<ndim; i++) {
+        r[i] = p.r[i];
+        v[i] = p.v[i];
+        a[i] = p.a[i];
```

Comment:
Is the acceleration needed here in this data structure?  Is it needed for example in the CD2010 viscosity?